docs/contributing/contributing-code/contributing-code-control-plane/running-controlplane-locally.md

```diff
@@ -21,8 +21,7 @@ The easiest way to get started is to launch Radius using VS Code. This will give
 If you need to manually test APIs you can reach them at the following endpoints after following these instructions:
 
 - UCP: port 9000
-- AppCore Namespace: port 8080
-- AppPortableResource Namespace: port 8081 (Applications.Datastores, Applications.Dapr and Applications.Messaging are hosted)
+- AppCore and AppPortableResource Namespaces: port 8080
```

Comment:
I dont think we call it AppCore anymore,  AppPortableResource isn't right either.  We could say applications-rp which serves all namespaces. Other suggestions welcome.

Reply:
Updated it to `Applications.Core RP / Portable Resources' Providers (applications-rp)` since this is the description used above